presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestExtractSpatialInnerJoin.java

```diff
@@ -165,8 +201,8 @@ private void testRadiusExpressionInDistanceQuery(String filter, String newFilter
                 .on(p ->
                         p.filter(PlanBuilder.expression(filter),
                                 p.join(INNER,
-                                        p.values(p.symbol("a"), p.symbol("name_a")),
-                                        p.values(p.symbol("b"), p.symbol("name_b"), p.symbol("r")))))
+                                        p.values(p.symbol("a", GEOMETRY), p.symbol("name_a")),
+                                        p.values(p.symbol("b", GEOMETRY), p.symbol("name_b"), p.symbol("r")))))
```

Comment:
These changes are needed because getExpressionType() is now called in tryCreateSpatialJoinByComparison(). If GEOMETRY is not specified here, their types will be somehow recognized as BIGINT by default and getExpressionType() will throw.